build/app_template_files/__NativeSwiftTemplateAppName__/Podfile

```diff
@@ -2,10 +2,21 @@
 # platform :ios, '8.0'
 
 target '__NativeSwiftTemplateAppName__' do
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/wmathurin/SalesforceMobileSDK-iOS-Specs.git'
```

Comment:
What does this mean for dependencies (e.g. CocoaLumberjack) that don't live in our specs repo?  Or do we envision adding local versions of those?  Or do they "just work" somehow?